config/default/FeatureFlagFormPropertyMapper.conf.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use oat\taoLti\models\classes\LtiProvider\FeatureFlagFormPropertyMapper;
+
+return new FeatureFlagFormPropertyMapper(
+    [
+        FeatureFlagFormPropertyMapper::OPTION_FEATURE_FLAG_FORM_FIELDS => []
+    ]
+);
```

Comment:
\n
also license

Reply:
Do we add licence to `conf` files?

Reply:
We have separate  folder called `headers` on the same level as `default` in configuration directory, that supposed to contain it  (historically)